commands/lily/watch.go

```diff
@@ -17,21 +20,25 @@ var LilyWatchCmd = &cli.Command{
 	Name:  "watch",
 	Usage: "Start, Stop, and List watches",
 	Subcommands: []*cli.Command{
+		LilyWatchCreateCmd,
 		LilyWatchStartCmd,
+		LilyWatchStopCmd,
+		LilyWatchListCmd,
```

Comment:
Not sure about this API. It's logical but when writing the design doc I thought we would do something like:

```
visor watch <-- starts a watch
visor walk <-- starts a walk
visor monitor-reorg <- starts monitoring reorgs

visor jobs list <-- lists all the of the above jobs
visor jobs stop <-- stops any of the jobs
visor jobs status <-- gets status of any job
```

I think the main difference is that there is a single way to see what jobs your visor node is running and to stop them if you need to. The API you have here would lose that.

BUT I am not convinced either way is better than others at the moment. What do you think?

Reply:
I'm in favor of your proposal -- it's also a smaller API surface, and easily extendable when/if more job types are added. I believe it will also align well with your suggestion below on [Task/Job/Scheduler](https://github.com/filecoin-project/sentinel-visor/pull/414#discussion_r596151173).